packages/react/package.json

```
@@ -216,7 +206,16 @@
   "peerDependencies": {
     "d3": "^7.0",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "@carbon/charts": ">=1.23.10 <2",
+    "@carbon/charts-react": ">=1.23.10 <2",
+    "@carbon/colors": ">=11 <12",
+    "@carbon/icons-react": ">=11 <12",
+    "@carbon/layout": ">=11 <12",
+    "@carbon/pictograms-react": ">=11 <12",
+    "@carbon/react": ">=1.88.0 <2",
+    "@carbon/themes": ">=11 <12",
```

Let's replace those with carets just to make sure we keep the majors but allow minor and patch updates
```suggestion
    "@carbon/charts": "^1.23.10",
    "@carbon/charts-react": "^1.23.10",
    "@carbon/colors": "^11.34.0",
    "@carbon/icons-react": "^11.61.0",
    "@carbon/layout": "^11.35.0",
    "@carbon/pictograms-react": "^11.69.0",
    "@carbon/react": "^1.88.0",
    "@carbon/themes": "^11.54.0",
```